h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -2973,7 +2973,7 @@ def cannaryHDFSTest(hdfs_name_node, file_name):
             return False
 
 
-def extract_scoring_history_field(aModel, fieldOfInterest):
+def extract_scoring_history_field(aModel, fieldOfInterest, grab1Ele=False):
```

Comment:
Please rename to "grabFirst" or "takeFirst". This one confused me a lot :)

Reply:
Sure